rest_framework_dataclasses/serializers.py

```diff
@@ -346,6 +348,21 @@ def build_relational_field(self, definition: DataclassDefinition, field_name: st
 
         return field_class, field_kwargs
 
+    def build_literal_field(self, definition: DataclassDefinition, field_name: str, type_info: TypeInfo) \
+            -> SerializerFieldDefinition:
+        """
+        Create fields for models.
+        """
+        field_class = rest_framework.fields.ChoiceField
+        choices = typing_utils.get_literal_choices(type_info.base_type)
+        field_kwargs = {
+            'choices': choices,
+            'allow_null': (None in choices) or type_info.is_optional,
+            'allow_blank': '' in choices,
```

Comment:
Does it hurt that the `None` and empty options are still in the `choices` list? It might be fine, I just haven't actually used `ChoiceField` ever, so I'm not too sure about its behaviour.

Reply:
Good catch, keeping `None` in the choices would make it also accept the string `'None'`, which is slightly unexpected. Better to omit it. :)

Keeping the empty string probably doesn't do anything bad, but I decided to omit it as well.